### ISSUE TITLE
Undebate: Update alt text for the Undebate image to adhere to WCAG

### ISSUE THREAD

**Issue by MattPereira** [hackforla/website#4040]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.
### Overview
As a developer, we need to provide clear and descriptive alt text for the Undebate image so that we adhere to the Web Content Accessibility Guidelines (WCAG) and we can achieve our mission of inclusivity.
### Action Items
- [x] Change the alt value within `_projects/undebate.md`:

**From:**
```
alt: 'Undebate with moderator and 4 participants, displayed question as "Why are you running for office?".'
```
**To:**
```
alt: 'Undebate'
```
**For PM, Merge Team, or Tech Lead**
- [x] Once the pull request associated with this issue is approved and merged, please update and edit epic #3866 by
  - [x] Checking off the dependency for this issue
  - [x] If all dependencies are checked off, please move issue to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
-  Undebate Page: https://www.hackforla.org/projects/undebate

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE:  `_projects/undebate.md`
- See the source file code on the repo: https://github.com/hackforla/website/blob/gh-pages/_projects/undebate.md
- This issue is part of epic #3866

#### Background information about the Jekyll framework (optional reading)
- [Jekyll Data Files](https://jekyllrb.com/docs/datafiles/)
- [Jekyll Front Matter](https://jekyllrb.com/docs/front-matter/)

#### Information about WCAG and alt text (optional reading)
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
- [WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)
- [Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)

**Comment by jefflueck** [comment]
i. Availability: 7:00 - 8:00 pm
ii. ETA: Finished.

**Comment by ExperimentsInHonesty** [comment]
@MattPereira it says "[MattPereira](https://github.com/MattPereira) moved this from QA to In progress (actively working) in [Project Board](https://github.com/hackforla/website/projects/7#card-88243297) [2 weeks ago](https://github.com/hackforla/website/issues/4040#event-8668574137)" on this issue but its closed.  Can you review and see what should happen?  I am putting it in the Questions column

**Comment by MattPereira** [comment]
@ExperimentsInHonesty 

Jeff, the assignee of this issue and a first time onboard, made the mistake of closing his issue manually before even creating a pull request, which triggered the bot moving the issue to QA column

I met Jeff a while back in my boot-camp days and was trying to help him understand the workflow

I forgot to re-open the issue after moving it that was my bad, but the PR was completed and merged so I think we are okay here.

**Comment by MattPereira** [comment]
Since I didn't re-open the issue the automation never kicked in to move issue out of `In Progress` column

Apologies for the confusion
